main.tf

```diff
@@ -1,13 +1,145 @@
-resource "random_integer" "example" {
+module "aws_config_label" {
+  source  = "cloudposse/label/null"
+  version = "0.21.0"
+
+  attributes = ["aws-config"]
+  context    = module.this.context
+}
+resource "aws_s3_bucket" "default" {
```

Comment:
Instead, let's use our `log-storage` module (and possibly incorporate anything missing upstream in that module)

Reply:
3c35b98330c7be15347afab22d46c26ee3389fb9